### ISSUE TITLE
Erroneous toast when using the video speed overlay button

### ISSUE THREAD

**Issue by KobeW50** [inotia00/ReVanced_Extended#1385]
### Type

Cosmetic

### Tools used

ReVanced Manager

### Application

Youtube 18.31.40

### Bug description

When you use the video speed overlay button to change the speed, a toast will appear saying that the default speed was changed even if you did not include the `Default Playback Speed` patch. The toast does not appear when changing the speed from the flyout menu.

It is cosmetic only so the default is not actually changed. (If I close the video and open a new one, the video speed will be reset)

The bug doesn't occur when you include the `Default Playback Speed` patch and disable the toggle in the Video settings.

### Steps to reproduce

1. Patch 18.31.40 with the Overlay buttons patch and without the Default Playback Speed patch.

2. Start a video and change the speed using the speed overlay button

### Relevant log output

```shell
Copying APK
Reading APK
Decoding app manifest
Loading patches
Merging integrations
Deleting existing resource cache directory
Decoding resources
Executing patches
Bypass ambient mode restrictions succeeded
Change homepage succeeded
Custom branding YouTube name succeeded
Custom branding icon MMT succeeded
Custom double tap length succeeded
Custom package name succeeded
Custom playback speed succeeded
Custom seekbar color succeeded
Default video quality succeeded
Disable QUIC protocol succeeded
Disable Shorts on startup succeeded
Disable auto captions succeeded
Disable haptic feedback succeeded
Disable hdr video succeeded
Disable landscape mode succeeded
Disable pip notification succeeded
Enable compact controls overlay succeeded
Enable debug logging succeeded
Enable external browser succeeded
Enable minimized playback succeeded
Enable new comment popup panels succeeded
Enable new splash animation succeeded
Enable new thumbnail preview succeeded
Enable old quality layout succeeded
Enable open links directly succeeded
Enable seekbar tapping succeeded
Enable tablet mini player succeeded
Enable tablet navigation bar succeeded
Enable time stamps speed succeeded
Enable wide search bar succeeded
Force OPUS codec succeeded
Force VP9 codec succeeded
Header switch succeeded
Hide account menu succeeded
Hide auto player popup panels succeeded
Hide autoplay button succeeded
Hide autoplay preview succeeded
Hide button container succeeded
Hide captions button succeeded
Hide cast button succeeded
Hide category bar succeeded
Hide channel avatar section succeeded
Hide channel watermark succeeded
Hide collapse button succeeded
Hide comment component succeeded
Hide crowdfunding box succeeded
Hide description components succeeded
Hide double tap overlay filter succeeded
Hide end screen cards succeeded
Hide end screen overlay succeeded
Hide feed flyout panel succeeded
Hide filmstrip overlay succeeded
Hide floating microphone succeeded
Hide fullscreen panels succeeded
Hide general ads succeeded
Hide handle succeeded
Hide info cards succeeded
Hide latest videos button succeeded
Hide layout components succeeded
Hide load more button succeeded
Hide mix playlists succeeded
Hide music button succeeded
Hide navigation buttons succeeded
Hide navigation label succeeded
Hide player button background succeeded
Hide player flyout panel succeeded
Hide player overlay filter succeeded
Hide previous next button succeeded
Hide quick actions succeeded
Hide seek message succeeded
Hide seekbar succeeded
Hide shorts components succeeded
Hide snack bar succeeded
Hide speed overlay succeeded
Hide suggested actions succeeded
Hide suggested video overlay succeeded
Hide suggestions shelf succeeded
Hide time stamp succeeded
Hide tooltip content succeeded
Hide trending searches succeeded
Hide video ads succeeded
Language switch succeeded
Layout switch succeeded
MicroG support succeeded
Optimize resource succeeded
Overlay buttons succeeded
Return YouTube Dislike succeeded
Settings succeeded
SponsorBlock succeeded
Spoof app version succeeded
Spoof player parameters succeeded
Swipe controls succeeded
Theme succeeded
Translations succeeded
Compiling modified dex files
Compiling modified resources
Signing APK
Patched
```

### Screenshots or videos

https://github.com/inotia00/ReVanced_Extended/assets/84587632/256c2ade-669e-4ab1-a842-b3f9f0dc14e4

### Device Environment

Samsung S22+
Android 13
One UI 5.1
Non-root

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] I have written the title and contents in English.
- [X] This issue does not reproduce on unpatched YouTube or YT Music.

**Comment by KobeW50** [comment]
Also inotia00, sorry that I forgot to warn you about deleting my reddit account. If you need to discuss anything you can open an issue in one of my repos or ping me anywhere on github. Or friend me on discord. Username is `kobew50`

**Comment by s7san007** [comment]
Another little cosmetic bug in YT Music. If you change video quality to "Auto", the toast will show "-2p". :)

![Screenshot_20230918_163053](https://github.com/inotia00/ReVanced_Extended/assets/130248949/7251ecb7-5599-4329-8df1-042fe6109a5b)

**Comment by girshcatt** [comment]
Stop, why your ytm toast has vanced icon?

**Comment by Spacellary** [comment]
@girshcatt I think they might be using the custom branding icon feature to make apps with custom icons :')

**Comment by girshcatt** [comment]
thanks for info

**Comment by KobeW50** [comment]
> @girshcatt I think they might be using the custom branding icon feature to make apps with custom icons :')
> 
> https://github.com/inotia00/ReVanced_Extended/issues/955#issuecomment-1536532265

Or cuynu's apks...

**Comment by Spacellary** [comment]
@KobeW50 Uh-uh 🚩

**Comment by s7san007** [comment]
> Or cuynu's apks...

Wdym? What apks?
I use RVX Manager 1.9.7 fork.

**Comment by inotia00** [comment]
fixed in [revanced-patches-v2.190.8](https://github.com/inotia00/revanced-patches/releases/tag/v2.190.8)
